### DISCUSSION TITLE
Is there a plan for the Next Release?

### DISCUSSION THREAD

**Discussion by gasscoelho** [AltimateAI/vscode-dbt-power-user#54] [Q&A] [answered]
Hi,

First of all, thank you so much for this awesome extension! It really helps and gives a real boost on developing with dbt. 

One little thing I missed was an option to generate the compiled code. I often prefer to use `dbt compile` when I'm debugging my model's code or trying some craze implementations 🤣

I noticed an open issue (#50) asking for this feature, so I would like to know when this feature will be coming to the extension or if you had plans for it.

Thanks!

**Comment by shindiogawa** [comment]
I agree with @gasscoelho. It's very helpfull to use dbt compile for debug purpose.

**Comment by mdesmet** [comment, marked as answer]
Hi, I have just released this feature: 

![image](https://user-images.githubusercontent.com/9265503/109432784-2e9aea00-7a0d-11eb-9788-a8fc5ec0cbaf.png)

Let us know if you like it and please rate the extension on https://marketplace.visualstudio.com/items?itemName=innoverio.vscode-dbt-power-user

**Reply by gasscoelho** [reply]
Thank you so much. That's really helpful!
